app/frontend/src/components/MogacoPost/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { useState, useEffect } from 'react';
+
+import { Input, Button, Textarea } from '@/components';
+
+import * as styles from './index.css';
+
+export function MogacoPostPage() {
+  const [date, setDate] = useState('');
+
+  useEffect(() => {
+    const today = new Date().toISOString().slice(0, 16);
```

Comment:
P3: 이 부분은 왜 useEffect()로 처리하셨는지 궁금합니다!

Reply:
그러게요. useState의 대표적인 좋지 않은 사용법이네요.
useState로 관리하는 date를 삭제하고 다음과 같이 변경하는 것이 좋겠어요!

```suggestion
    const date = new Date().toISOString().slice(0, 16);
```

Reply:
앗 수정해야했는데 빼먹었네요 수정하겠습니다!

Reply:
https://github.com/boostcampwm2023/web17_morak/pull/137/commits/b981125d05f5a59a7e8d624996dd2e73175db70c
수정하였습니다